benchmarks/src/jvmMain/kotlin/SegmentPoolBenchmarkMT.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+package kotlinx.io.benchmarks
+
+import kotlinx.io.*
+import kotlinx.benchmark.*
+import org.openjdk.jmh.annotations.Group
+import org.openjdk.jmh.annotations.GroupThreads
+
+@State(Scope.Benchmark)
+open class SegmentPoolBenchmarkMT {
+    private fun testCycle(): Buffer {
+        val buffer = Buffer().writeByte(0)
```

Comment:
Maybe we should add cycles here to increase the probability of concurrency access to segments?

However, in this case, the test time may become unstable.

Reply:
Under the hood, a runner invokes benchmarked methods in a loop, so I don't think it'll have an impact on overall benchmark execution.